src/containers/PermissionsApplication.jsx

```diff
@@ -110,6 +114,18 @@ const PermissionsApplication = ({ t }) => {
           </IconButton>
           <NavigationList>
             <PageTitle>{appName.toUpperCase()}</PageTitle>
+            <CircleButton
+              label={t('Permissions.open')}
+              href={
+                isApp(matchingQueryResult.data[0].type)
+                  ? matchingQueryResult.data[0].links.related
+                  : window.location.href
+                      .replace('settings', 'home')
+                      .replace('permissions/slug', 'connected')
```

Comment:
If you need to make a link between cozy's apps, you should always use AppLinker from Cozy-UI.

You'll find exemple of how to use it in the Cozy code base, but you'll need to use generateWebLink, and to use it, you'll have to get the subdomainType. And to get the subdomainType, you'll need to read it from the client: `  const { subdomain: subDomainType } = client.getInstanceOptions()`

Reply:
I tried to do the same than in [this file](https://github.com/cozy/cozy-store/blob/000c8838f0e6ebc6e5b9e04118594508db169248/src/ducks/apps/components/ApplicationPage/Header.jsx#) from cozy-store (same open app button). It works for opening an app, but for the [konnector](https://github.com/cozy/cozy-store/blob/000c8838f0e6ebc6e5b9e04118594508db169248/src/ducks/apps/components/ApplicationPage/Header.jsx#L47) the thing is that I need the `related` link. I have it for apps but not for konnectors.

Reply:
Here `matchingQueryResult.data[0]` should contains all data needed for the AppLinker to work. Then in adition to generateWebLink you should be able to do something like this (the code may not work as is) :

```js
// this contains all the app/connectors data (the minimum for us is to have the slug)
const app = matchingQueryResult.data[0]

// this allows to generate the full absolute href link
const { subdomain: subDomainType } = client.getInstanceOptions()
const homeHref = generateWebLink({
    cozyUrl: client.getStackClient().uri,
    'home',
    subDomainType
  })

// this allows to create a component that do its navigation based on its context (in webapp or in mobile app)
return <AppLinker app={app} href={homeHref}>
  {({ onCLick, href }) => (
    // this is your component as it was in your PR (don't forget to add the onClick)
    <CircleButton
      label={t('Permissions.open')}
      href={href}
      onClick={onClick}
    >
      <Icon icon={OpenappIcon} />
    </CircleButton>
  )}
</AppLinker>
```

Reply:
@Ldoppea this code will work for an `app` not for a konnector since they don't "own" a specific URL.

For a konnector, we can use: 
```jsx
// this allows to generate the full absolute href link
const { subdomain: subDomainType } = client.getInstanceOptions()
const homeHref = generateWebLink({
    cozyUrl: client.getStackClient().uri,
    slug: 'home',
    subDomainType,
    nativePath: `/#/connected/{appOrKonnector.slug}`
  })

Reply:
@vdnj : could you add one guideline about not using window.location in our cozy-guideline please? (after you finish this PR)